src/WebJobs.Script.WebHost/DefaultScriptHostBuilder.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.Azure.WebJobs.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public sealed class DefaultScriptHostBuilder : IScriptHostBuilder
```

Comment:
Introducing the implementation and interface to make testing a little easier for the hosted service. This was pretty cumbersome to setup.